.github/workflows/ci.yml

```diff
@@ -24,18 +40,16 @@ jobs:
         with:
           path: ${{ env.FOREMAN_PLUGIN_NAME }}
       - name: Setup Ruby
-        uses: actions/setup-ruby@v1
+        uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby-version }}
       - name: Setup Node
         uses: actions/setup-node@v1
         with:
           node-version:  ${{ matrix.node-version }}
       - name: Setup Foreman Plugin
-        uses: kamils-iRonin/setup-foreman-plugin@master
+        uses: kamils-iRonin/setup-foreman-plugin@install_plugin
```

Comment:
waiting for https://github.com/kamils-iRonin/setup-foreman-plugin/pull/4 to be merged